src/pages/HomePage.tsx

```diff
@@ -1,10 +1,12 @@
 import BaseLayout from "@/layouts/BaseLayout";
+import { FeaturedProductView } from "@/components/Product/FeaturedProductView/FeaturedProductView";
 
 const HomePage = () => {
   return (
     <BaseLayout>
-      <div className="flex h-full grow items-center justify-center bg-slate-100 p-10">
+      <div className=" bg-slate-100 p-10">
         <h1>Home Page</h1>
+        <FeaturedProductView></FeaturedProductView>
```

Comment:
Cuando no tiene hijos puedes hacer 
```suggestion
        <FeaturedProductView />
```